jwala-services/src/main/java/com/cerner/jwala/service/resource/impl/ResourceServiceImpl.java

```diff
@@ -1003,6 +1023,37 @@ private void doUnpack(final String hostName, final String destPath) {
         return resourceFileString;
     }
 
+    private Application setApplicationWarDeployPath(Application application) {
```

Comment:
I like the idea of the fluent interface, but in the interest of time I just extracted some private methods and added a java doc. In the future I will definitely think more "fluently" :)